src/components/Table/hoc/withTableActions/withTableActions.tsx

```diff
@@ -121,6 +122,7 @@ export function withTableActions<I extends TableDataItem, E extends {} = {}>(
                         disabled={disabled}
                         className={BUTTON_CLASSNAME}
                         onClick={this.handleActionsButtonClick.bind(this, {item, index})}
+                        size={actionsButtonSize}
```

Comment:
Besides button there is a DropdownMenu that should be scaled accordingly. Sizes should match.